app/models/application_form.rb

```diff
@@ -265,7 +265,7 @@ def blank_application?
   end
 
   def candidate_has_previously_applied?
```

Comment:
Shouldn't this method have a different name now? `candidate_has_previously_submitted_application?` or something similar.

Reply:
I don't think it does. Have you applied if you haven't submitted an application?

Reply:
I guess not, I'll mark it as resolved. I would have been clearer from the title what it checks for.